src/components/MoneyRequestHeader.tsx

```diff
@@ -159,51 +245,36 @@ function MoneyRequestHeader({report, parentReportAction, policy, onBackButtonPre
                 shouldDisplaySearchRouter={shouldDisplaySearchRouter}
                 onBackButtonPress={onBackButtonPress}
             >
-                {shouldShowMarkAsCashButton && !shouldUseNarrowLayout && (
-                    <Button
-                        success
-                        text={translate('iou.markAsCash')}
-                        style={[styles.p0]}
-                        onPress={markAsCash}
-                    />
-                )}
-                {isDuplicate && !shouldUseNarrowLayout && (
-                    <Button
-                        success
-                        text={translate('iou.reviewDuplicates')}
-                        style={[styles.p0, styles.ml2]}
-                        onPress={() => {
-                            if (!reportID) {
-                                return;
-                            }
-                            Navigation.navigate(ROUTES.TRANSACTION_DUPLICATE_REVIEW_PAGE.getRoute(reportID, Navigation.getReportRHPActiveRoute()));
-                        }}
-                    />
+                {!shouldUseNarrowLayout && (
+                    <View style={[styles.flexRow, styles.gap2]}>
+                        {!!primaryAction && primaryActionOptions[primaryAction]}
+                        {!!secondaryActions.length && (
+                            <ButtonWithDropdownMenu
+                                success={false}
+                                onPress={() => {}}
```

Comment:
so far it's just place holder right?

Reply:
@sumo-slonik this button only should show more actions that are defined above. Clicking on one of the options performs actual action:

https://github.com/user-attachments/assets/02af319a-d3ae-4ffd-8de6-9df12d869607